pkg-manager/plugin-commands-installation/test/install.ts

```diff
@@ -52,3 +53,124 @@ test('install with no store integrity validation', async () => {
 
   expect(fs.readFileSync('node_modules/is-positive/readme.md', 'utf8')).toBe('modified')
 })
+
+describe('install injected dependencies', () => {
```

Comment:
Woops didn't mean to include with this PR (https://github.com/pnpm/pnpm/pull/7048) but hopefully serves as proof of passing tests...